stock_inventory_cost_info/tests/test_stock_inventory_cost_info.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestStockInventoryCostInfo(TransactionCase):
+    def setUp(self, *args, **kwargs):
+        super(TestStockInventoryCostInfo, self).setUp(*args, **kwargs)
+        self.product_4 = self.env.ref("product.product_product_4")
+        self.product_4.write({'standard_price': 1000})
+        self.product_5 = self.env.ref("product.product_product_5")
+        self.product_5.write({'standard_price': 2000})
+        # Initial inventory: set quantities to zero
+        initial_inventory = self.env['stock.inventory'].create({
+            'name': 'Initial inventory',
+            'filter': 'partial',
+            'line_ids': [(0, 0, {
+                'product_id': self.product_4.id,
+                'location_id': self.env.ref('stock.warehouse0').lot_stock_id.id
+            }), (0, 0, {
+                'product_id': self.product_5.id,
+                'location_id': self.env.ref('stock.warehouse0').lot_stock_id.id
+            })]
+        })
+        initial_inventory.action_reset_product_qty()
+        initial_inventory.action_done()
+        # Another inventory adjustment
+        self.inventory = self.env['stock.inventory'].create({
+            'name': 'Another inventory',
+            'filter': 'partial',
+        })
+        initial_inventory.action_start()
+        self.inventory.write({
+            'line_ids': [(0, 0, {
+                'product_id': self.product_4.id,
+                'product_qty': 10,
+                'location_id': self.env.ref('stock.warehouse0').lot_stock_id.id
+            }), (0, 0, {
+                'product_id': self.product_5.id,
+                'product_qty': 20,
+                'location_id': self.env.ref('stock.warehouse0').lot_stock_id.id
+            })]
+        })
+
+    def test_compute_adjustment_cost(self):
+        """Tests if the adjustment_cost is correctly computed."""
+        lines = self.inventory.line_ids
+        line = lines.filtered(lambda r: r.product_id == self.product_4)
+        self.assertEqual(line.adjustment_cost, 10000)
+        line = lines.filtered(lambda r: r.product_id == self.product_5)
+        self.assertEqual(line.adjustment_cost, 40000)
```

Comment:
Change the product cost and validate the inventory for checking the new adjustment_cost